app/javascript/oldjs/application.js

```diff
@@ -30,8 +30,3 @@ require('./miq_list_grid.js');
 require('./miq_toolbar.js');
 require('./miq_c3.js');
 require('./miq_explorer.js');
-
-if (process.env.NODE_ENV === 'development' && process.env.CYPRESS !== 'true') {
-  require('./miq_debug.js');
-  require('./miq_debug.css');
-}
```

Comment:
TODO: this conditional doesn't work correctly as CYPRESS env variable is never defined.  I tried REACT_APP_CYPRESS and other things such as trying to push it into webpack but nothing worked so for now, I have this removed in this PR as the notifications cause far too many sporadic test failures.